modules/002-deckhouse/templates/deployment.yaml

```
@@ -26,10 +26,11 @@ spec:
     kind: Deployment
     name: deckhouse
   updatePolicy:
-    updateMode: "Initial"
+    updateMode: "Auto"
```

```suggestion
    updateMode: "Initial"
```

no reason to evict a pod. Deckhouse will not migrate to a new node